### PR TITLE
426 error modal for round metadata

### LINKED ISSUE #426: application: add error modal for when round metadata isn't loading
**User Story**
As a project owner 
I want to know if there has been an error loading the round application 
so that I what next steps I should take 

**Acceptance Criteria**
GIVEN that there is an error loading the round application
WHEN I look at the page
THEN I see a modal that informs me of error details 
AND what next steps I should take (i.e. refresh page, contact support) 

GIVEN that I see the error modal 
WHEN I click on "Close" 
THEN I am redirected to the Grant Hub home page (or "My Projects" page) 

GIVEN that I see the error modal 
WHEN I click on "Refresh" 
THEN the page is reloaded

**Notes/Assumptions**: 
* N/A

**Product & Design Links**:  
* [Figma link](https://www.figma.com/file/obBzCCL8KwoTlVBb5g3ccN/Grant-Hub---Wireframes?node-id=2557%3A17537)

Screen Shot 2022-10-02 at 3 01 44 PM

**Tech Details**: 
* N/A 

**Open Questions**: 
* N/A

### PR DESCRIPTION
**Pull request by ghostffcode**
Sample error when I use an invalid round address:

Screenshot 2022-11-03 at 5 41 02 PM

### CODE REVIEW

**Review by michellema1208 — APPROVED** [review]

File: client/src/components/rounds/Apply.tsx [hunk 1]
```diff
@@ -139,15 +140,61 @@ function Apply() {
   }, [props.applicationStatus, props.applicationError]);
 
   if (props.roundStatus === RoundStatus.Error) {
-    return <div>Error loading round data: {props.roundError}</div>;
+    // return <div>Error loading round data: {props.roundError}</div>;
```
Comment (DanieleSalatti) [inline client/src/components/rounds/Apply.tsx]: Remove?

File: client/src/components/base/VerificationForm.tsx [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import { RootState } from "../../reducers";
-
+import Button, { ButtonVariants } from "./Button";
+/*
 const initialFormValues = {
   github: "",
   twitter: "",
 };
-
+*/
```
Comment (DanieleSalatti) [inline client/src/components/base/VerificationForm.tsx]: I think we can remove this (and the commented out code on line 31)

**Review by DanieleSalatti — APPROVED** [review]
LGTM

Just left some minor nits on commented out code

File: client/src/components/rounds/Apply.tsx (hunk 1 above)
Comment (ghostffcode) [reply client/src/components/rounds/Apply.tsx]: Yeah, thanks

**Pull request merged**